### ISSUE TITLE
UI: remove progress bars? 

### ISSUE THREAD

**Issue by durkie** [PublicDesignWorkshop/FoodParent#9]
They make me perceive the site as slow because it's drawing attention to the fact that it seems like something is always loading.

**Comment by ThePlushEater** [comment]
I removed, but I think we need to think about how to provide feedback if the connection has failed.

**Comment by durkie** [comment]
agreed -- if something fails let's show a message, but i think it looks better without the progress bars. thanks! :+1:
